ballerina-tests/tests/records.bal

```diff
@@ -229,3 +229,14 @@ public enum InstrumentType {
     # Instruments with leather, wooden or metal surfaces
     PERCUSSION
 }
+
+# Represents the standard WebSocket message structure for subscriptions
+#
+# + 'type - Type of the WebSocket message  
+# + id - Operation id for uniquely identifying server responses and connecting them with the client's requests  
+# + payload - The requested data to go with the specific message type
+public type WSPayload record {|
+    string 'type;
+    string id?;
+    json payload?;
+|};
```

Comment:
Can't we use the same `WSPayload` type defined in `records.bal` since it's a public type?

Reply:
How did you try this? `graphql:WSPayload` should be worked.

Reply:
Ah. forgot to use it.